deployments/kubernetes/chart/reloader/values.yaml

```diff
@@ -11,6 +11,26 @@ reloader:
   deployment:
     nodeSelector:
     # cloud.google.com/gke-nodepool: default-pool
+
```

Comment:
Kindly add these values(affinity, tolerations and resources) in `/deployments/kubernetes/templates/chart/values.yaml.tmpl` file too!